### PR TITLE
feat: more comprehensive React docs

### PR DESCRIPTION
**Pull request by bangarang**

### CODE REVIEW

File: apps/embedding/react/components.mdx [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+        return record;
+      },
+    ],
+  ]}
+/>
+```
+
+### Sheet
+
+Similar to the Simplified SDK Approach
```
Comment (elisadinsmore) [inline apps/embedding/react/components.mdx:91]: Maybe 
> The `Sheet` option is similar to the [Simplified SDK Approach](apps/embedding/reference/simple).

File: apps/embedding/react/components.mdx [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+
+Configures a Sheet config, with an onRecordHook helper
```
Comment (elisadinsmore) [inline apps/embedding/react/components.mdx:93]: > This option configures a Sheet config with an onRecordHook helper.

File: apps/embedding/react/components.mdx [hunk 3]
```diff
@@ -0,0 +1,350 @@
...
+
+Note: an `onSubmit` can be added to the Workbook Component for ease of use
```
Comment (elisadinsmore) [inline apps/embedding/react/components.mdx:95]: Would it be beneficial to call out in some location where they would want to use onSubmit vs where they wouldn't? I don't expect people who come right to this guide to know where to look for that information.

File: apps/embedding/react/components/FlatfileProvider.mdx [hunk 4]
```diff
@@ -0,0 +1,45 @@
+---
+title: FlatfileProvider Component
```
Comment (elisadinsmore) [inline apps/embedding/react/components/FlatfileProvider.mdx:2]: This page needs to be added to the sidebar options

File: apps/embedding/react/components.mdx [hunk 5]
```diff
@@ -0,0 +1,350 @@
...
+
+Configures all Workbook Creation request data and an new onRecordHooks helper function
+
+Sheets can be passed along in the `config`, `onSubmit` can be added, along with a new `onRecordHooks` helper.
+`onRecordHooks` takes an array of record hooks. Each one can take a slug for manually setting each per sheet. Otherwise, if no slug is added, it will apply the record hook to the corresponding sheet in the Workbook index.
+
+```tsx
+<Workbook
+  config={workbook}
+  onSubmit={(sheet) => {
```
[the remark this answers is not in the document]
Comment (bangarang) [reply apps/embedding/react/components.mdx:74]: They will all just add to the Workbook Actions - which does get a bit crowded.

File: apps/embedding/react/components.mdx [hunk 6]
```diff
@@ -0,0 +1,350 @@
...
+>
+      <FFApp />
+</FlatfileProvider>
+```
+
+### Space
+
+Configures all Space Creation request data and the `id` for reusing a space
+
+```tsx
```
[the remark this answers is not in the document]
Comment (bangarang) [reply apps/embedding/react/components.mdx:39]: Definitely should!

**Review by carlbrugger — APPROVED** [review]

File: apps/embedding/react/components.mdx [hunk 7]
```diff
@@ -0,0 +1,367 @@
...
+      },
+    },
+  }}
+/>
+```
+
+When you want to re-use a space, you'll need to pass an `accessToken` to the `FlatfileProvider`, then this is where you'll add the `id` of the Space you want to re-use.
+
+```tsx
+const FFApp = () => <Space config={{ id: "us_sp_123456" }} />;
```
Comment (bangarang) [inline apps/embedding/react/components.mdx:62]:
```suggestion
const FFApp = () => <Space id={"us_sp_123456" } />;
```

**Pull request merged**